examples/handling/interface/polygonize.py

```diff
@@ -0,0 +1,40 @@
+"""
+Polygonize a raster
+===================
+
+This example demonstrates the polygonizing of a raster using :func:`geoutils.Raster.polygonize` and :func:`geoutils.Mask.polygonize`.
+"""
+# sphinx_gallery_thumbnail_number = 3
```

Comment:
This line gets rendered, it probably shouldn't be?

Reply:
Solved in [9d487d0](https://github.com/GlacioHack/geoutils/pull/351/commits/9d487d0a85fe2acffbeb4b75d2e3765d23090f66)